internal/namespaces/config/commands.go

```diff
@@ -29,10 +33,57 @@ func GetCommands() *core.Commands {
 }
 
 func configRoot() *core.Command {
+	configPath := scw.GetConfigPath()
+	envVarTable := bytes.Buffer{}
+	w := tabwriter.NewWriter(&envVarTable, 5, 1, 2, ' ', tabwriter.ANSIGraphicsRendition)
+	for _, envVar := range [][2]string{
+		{"SCW_ACCESS_KEY", "The access key of a token (create a token at https://console.scaleway.com/account/credentials)"},
+		{"SCW_SECRET_KEY", "The secret key of a token (create a token at https://console.scaleway.com/account/credentials)"},
+		{"SCW_DEFAULT_ORGANIZATION_ID", "The default organization ID"},
+		{"SCW_DEFAULT_REGION", "The default region"},
+		{"SCW_DEFAULT_ZONE", "The default availability zone"},
+		{"SCW_API_URL", "URL of the API"},
+		{"SCW_INSECURE", "Set this to true to enable the insecure mode"},
+		{"SCW_PROFILE", "Set the config profile to use"},
+	} {
+		fmt.Fprintf(w, "  %s\t%s\n", terminal.Style(envVar[0], color.Bold, color.FgBlue), envVar[1])
+	}
+	w.Flush()
 	return &core.Command{
-		Short:     `Config file management`,
-		Long:      `Manage your Scaleway CLI config file.`,
+		Short: `Config file management`,
+		Long: `Config management engine is common across all Scaleway developer tools (CLI, terraform, SDK, ... ). It allows to handle Scaleway config through two ways: environment variables and/or config file.
```

Comment:
Can you try with [RemoveIndent](https://github.com/scaleway/scaleway-cli/blob/v2/internal/interactive/print.go#L66)?